content/en/real_user_monitoring/browser/data_collected.md

```diff
@@ -267,6 +267,16 @@ Source errors include code-level information about the error. For more informati
 | `action.frustration.type:rage_click`  | string | The rage clicks detected by the RUM Browser SDK.              |
 | `action.frustration.type:error_click` | string | The error clicks detected by the RUM Browser SDK.             |
 
+### UTM attributes
+
+| Field                | Type   | Description                                                   |
```

Comment:
For consistency, recommend adding a period to all the descriptions.

Reply:
sure, will add